hermes-benchmark/src/jmh/java/pl/allegro/tech/hermes/benchmark/environment/HermesServerFactory.java

```diff
@@ -54,8 +56,11 @@ static HermesServer provideHermesServer() throws IOException {
                 .overrideProperty(FRONTEND_TOPIC_METADATA_REFRESH_JOB_ENABLED, false)
                 .overrideProperty(FRONTEND_GRACEFUL_SHUTDOWN_ENABLED, false);
         Trackers trackers = new Trackers(Collections.emptyList());
+        TrackingHeadersExtractor trackingHeadersExtractor = new DefaultTrackingHeaderExtractor(headers -> headers);
```

Comment:
just asking, why is there `headers -> headers` in constructor?

Reply:
This class is used for extracting tracking headers. It relies on `HeadersPropagator` because it is using part of its functionality for filtering headers. This setup is only needed for hermes benchmarking, so we are giving here only simple implementation which just returns all the headers without any filtering.